### ISSUE TITLE
Question about the bot and the backtesting

### ISSUE THREAD

**Issue by Kaidokuni** [freqtrade/freqtrade#2203]
1-The bot is only configured to take long position, right? But can it take short position too? (Never mind, I just saw it was the subject of the #2183 ) 
2- I saw a post that wanted to increase the "stake_amount" to each new trade ( #2181 ), but can not the bot also use the leverage?* 
3-When I backtested my strategy, I set the "trailing_only_offset_is_reached" parameter to "false" and changed the value of my "trailing_stop_positive_offset" to see if it would change something in my result, and it did, why? If it's set to false, the "trailing_stop_positive" should be the only one  active, right?
4- When backtesting the strategy, the fees can be/or are included? I wonder if this is what the 0.0005 stand for in the hyperopt sharpe ratio ?(total_profit = total_profit - 0.0005)
5- In the result of the backtest, the value for the total profit is still the same despite the fact that I change the "stake_currency" (in my case, I switch "BTC" to "USDT"), so should I consider BTC as the default value for the backtest?

**Comment by xmatthias** [comment]
Well you answered 1 yourself

2) no, ccxt does not unify that, and implementing it on a "per exchange" basis would be too much maintenance at this point.
3) best read the [documentation](https://www.freqtrade.io/en/latest/stoploss/#custom-positive-loss) regarding this point - this should clarify questions regarding that
4) fees are included and use the "basic"/default exchange fees -> does not inclued fee reductions for very active traders or such.
Sharpe-ratio considers "slippage", as commented in the comment right before that line: `# adding slippage of 0.1% per trade`

5) when you switch stake-currency, i assume you've also switched all whitelisted pairs?
  Total profit always refers to BTC - since BTC is (probably) considered as a reference cryptocurrency (we maybe should change this sometime in the future to show profits in stake currency instead ... but it's not a bug, it's currently working as it was designed)

**Comment by Kaidokuni** [comment]
5- Yes, I added the 5 USDT pairs with the most volume, but as I have also kept some BTC pairs, I wonder how the bot will behave when  the dry_run will be disabled, because I can't add an other currency in "stake-currency" (""USDT", "BTC"" didn't work)
thanks for the rest

**Comment by hroff-1902** [comment]
One instance of the bot can trade only one currency, defined in the stake_currency setting. All pairs in the white- and blacklist shall be pairs with stake currency specified as quote currency. If you want to trade pairs with multiple quote currencies you should run multiple instances of the bot with different configs (where corresponding stake_currency is set).

**Comment by Kaidokuni** [comment]
So I have to create another configuration file containing only the pairs with the same quote currencies as the stake currency, right? Is there a reason for this? Allowing the bot to have multiple stake currencies in one configuration file isn't more simple?

**Comment by xmatthias** [comment]
it may seem like that would be simpler, but really it isn't once you consider market dynamics.

Since the markets themselvs differ strongly (XRP/ETH does not behave the same as XRP/BTC) you'll probably need different strategies per stake currency too (I've yet to see a strategy that works reliably for both ETH and BTC trading).

Then it becomes difficult - you have 2 stake currencies, and 2 strategies (one for each stake currency)- maybe you need a 3rd strategy for low-value pairs (since they again need to be trade differently...).

Combining them all and accounting for all of these "special cases" would complicate the configuration a lot - so running 2 bots (one per stake currency) is much simpler, for both the user, and from a maintainability standpoint.

**Comment by hroff-1902** [comment]
> So I have to create another configuration file containing only the pairs with the same quote currencies as the stake currency, right? Is there a reason for this? Allowing the bot to have multiple stake currencies in one configuration file isn't more simple?

No, support for multiple stake_currency'es would be a non-simple change with current freqtrade legacy design. If you have python programming skills, you can join and review the codebase.

**Comment by Kaidokuni** [comment]
Okay I understand better, thank you for the clarification. Regarding the strategy, do you have criteria to meet before being confident to launch it? For example a hyperopt above a certain value, a backtest with a certain average profit (or total profit) or leaving it for a certain length of time with the dry running mode enabled?

**Comment by hroff-1902** [comment]
I suggest to discuss this kind of questions with other users at the freqtrade Slack channel.
